Documentation/dev-guide/local_cluster.md

```diff
@@ -1,90 +1,152 @@
 
 The started etcd member listens on `localhost:2379` for client requests.
 
-To interact with the started cluster by using etcdctl:
+### Interacting with cluster
 
-```
-# use API version 3
-$ export ETCDCTL_API=3
+Use `etcdctl` to interact with the started cluster:
 
-$ ./etcdctl put foo bar
-OK
+1. Set the environment variable to `ETCDCTL_API=3` so that `etcdctl` will use the v3 client instead of the default v2.
 
-$ ./etcdctl get foo
-bar
-```
+    ```
+    # use API version 3
+    $ export ETCDCTL_API=3
+    ```
+2. Store an example key-value pair in the cluster:
+
+    ```
+      $ ./etcdctl put foo bar
+      OK
+    ```
+    If OK is printed, storing key-value pair is successful.
+
+3. Fetch the value of the key, `foo` that is stored in the previous step:
+
+    ```
+    $ ./etcdctl get foo
+    bar
+    ```
+    If `bar` is returned, interaction with the etcd cluster is working as expected.
 
 ## Local multi-member cluster
 
 A `Procfile` at the base of this git repo is provided to easily set up a local multi-member cluster. To start a multi-member cluster go to the root of an etcd source tree and run:
 
-```
-# install goreman program to control Profile-based applications.
-$ go get github.com/mattn/goreman
-$ goreman -f Procfile start
-...
-```
+### Starting the cluster
 
-The started members listen on `localhost:2379`, `localhost:22379`, and `localhost:32379` for client requests respectively.
+1. Install `goreman` program to control Procfile-based applications:
 
-To interact with the started cluster by using etcdctl:
+    ```
+    $ go get github.com/mattn/goreman
+    ```
+2. Start the `goreman` program:
 
-```
-# use API version 3
-$ export ETCDCTL_API=3
-
-$ etcdctl --write-out=table --endpoints=localhost:2379 member list
-+------------------+---------+--------+------------------------+------------------------+
-|        ID        | STATUS  |  NAME  |       PEER ADDRS       |      CLIENT ADDRS      |
-+------------------+---------+--------+------------------------+------------------------+
-| 8211f1d0f64f3269 | started | infra1 | http://127.0.0.1:2380  | http://127.0.0.1:2379  |
-| 91bc3c398fb3c146 | started | infra2 | http://127.0.0.1:22380 | http://127.0.0.1:22379 |
-| fd422379fda50e48 | started | infra3 | http://127.0.0.1:32380 | http://127.0.0.1:32379 |
-+------------------+---------+--------+------------------------+------------------------+
-
-$ etcdctl put foo bar
-OK
-```
+    ```
+    $ goreman -f Procfile start
+    ```
 
-To exercise etcd's fault tolerance, kill a member:
+The started members listen on `localhost:2379`, `localhost:22379`, and `localhost:32379` respectively for client requests.
 
-```
-# kill etcd2
-$ goreman run stop etcd2
+### Interacting with cluster
 
-$ etcdctl put key hello
-OK
+Use `etcdctl` to interact with the started cluster:
 
-$ etcdctl get key
-hello
+1. Set the environment variable to `ETCDCTL_API=3` so that `etcdctl` will use the v3 client instead of the default v2.
```

Comment:
Set the environment to have `ETCDCTL_API=3` so `etcdctl` uses the v3 API instead of defaulting into v2.